sentry-kotlin-multiplatform/src/androidMain/kotlin/io/sentry/kotlin/multiplatform/SentryScope.kt

```diff
@@ -0,0 +1,114 @@
+package io.sentry.kotlin.multiplatform
+
+import io.sentry.ScopeCallback
+import io.sentry.kotlin.multiplatform.extensions.*
+
+actual class SentryScope : ISentryScope {
+
+    // We are directly modifying the Android SDK scope
+    private var scope: AndroidSentryScope? = null
```

Comment:
This should not be nullable I guess, See https://github.com/getsentry/sentry-kotlin-multiplatform/pull/24/files#r935254139